vtexdocs/devportal
# Feat/api feedback

#### What is the purpose of this pull request?

To make the "Was this Helpful?" component functional.

#### What problem is this solving?

- The information sent by the user was not stored.
- When the feedback modal opened the page scrolled to the top.
- The feedback component stayed the same across all documentations instead of resetting.

Further information can be found [here](https://www.notion.so/vtexhandbook/Componente-Was-this-helpful-funcional-f21591656c67495596c49217377e453a?pvs=4#c4244630a94946c9a421dc584bc7aa6b)

#### How should this be manually tested?

Open the [preview](https://deploy-preview-296--elated-hoover-5c29bf.netlify.app/), use the feedback component at the end of a documentation page and check if the feedback was stored in the designated [spreadsheet](https://docs.google.com/spreadsheets/d/1nXE0Rb7b50ne7Ty33595yFg781NmFTLHB7ZYHUCe1_g/).

The spreadsheet should show the timestamp, the documentation page, the type of feedback (positive or negative) and the comment added by the user (it can be empty).

#### Screenshots or example usage

#### Types of changes
